src/go/k8s/pkg/resources/configmap.go

```diff
@@ -0,0 +1,137 @@
+package resources
+
+import (
+	"context"
+
+	redpandav1alpha1 "github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	seedSuffix	= "-seed"
+	dataDirectory	= "/var/lib/redpanda/data"
+)
+
+type ConfigMapResource struct {
```

Comment:
NIT: Should we create compile guard `var _ OwnedResource = ConfigMapResource{}`?